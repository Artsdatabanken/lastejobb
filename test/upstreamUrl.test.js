const { git } = require("lastejobb");

test("Schema url", () => {
  const actual = git.upstreamUrlForFile("./schema.json");
  expect(actual).toMatchSnapshot();
});
