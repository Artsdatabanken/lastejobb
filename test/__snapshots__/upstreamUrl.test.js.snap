// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schema url 1`] = `"https://raw.githubusercontent.com/Artsdatabanken/lastejobb/master/schema.json"`;
